persistence/src/test/java/com/epam/rft/atsy/persistence/repositories/ApplicationsRepositoryIT.java

```diff
@@ -86,24 +88,38 @@ public void findByCandidateEntityShouldFindThreeApplicationForCandidateWithThree
         .id(4L)
         .name("facebook")
         .build();
+    ChannelEntity expectedFourthChannelEntity = ChannelEntity.builder()
+        .id(6L)
+        .name("linkedin adatbázis")
+        .build();
     PositionEntity expectedPositionEntity = PositionEntity.builder()
         .id(1L)
         .name("Fejlesztő")
         .build();
     Date nearNow = currentDateMinus(5);
 
+    SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+    Date expectedDate = null;
+    try {
+      expectedDate = simpleDateFormat.parse("2016-07-26 11:48:55");
+    } catch (ParseException e) {
+      e.printStackTrace();
```

Comment:
Never use printStackTrace.

In test usually you don't need to handle exceptions, if an exception occurs, than your test case will fail with error.